src/table/table-role/grid-navigation.tsx

```diff
@@ -70,8 +74,8 @@ class GridNavigationProcessor {
 
   // State
   private focusedCell: null | FocusedCell = null;
-  private focusables = new Set<FocusableDefinition>();
-  private focusHandlers = new Map<FocusableDefinition, FocusableChangeHandler>();
+  private focusables = new Map<Element, boolean>();
```

Comment:
What is the advantage of using a Map of boolean here vs a Set?

Reply:
The `this.focusables` property keeps all elements that were registered and their state (focusable / not focusable). Can't use a set because it is necessary to iterate over focusables to issue updates or do a cleanup.